src/server/project.ts

```diff
@@ -212,7 +213,7 @@ namespace ts.server {
             readonly projectService: ProjectService,
             private documentRegistry: DocumentRegistry,
             hasExplicitListOfFiles: boolean,
-            languageServiceEnabled: boolean,
+            exceededFilename: string | undefined,

```

Comment:
Verify this doesnt break current API users

Reply:
I didn't found any place where we expose this constructor. I think API consumers should be fine with the change. Let me know if you think otherwise.